hyperlane/relayer.go

```diff
@@ -264,6 +269,11 @@ func (r *relayer) isRelayFeeLessThanMax(
 ) (bool, error) {
 	txFeeUUSDC, err := r.hyperlane.QuoteProcessUUSDC(ctx, domain, message, metadata)
 	if err != nil {
+		if strings.Contains(err.Error(), "execution reverted") {
```

Comment:
hate that there's no good way to get an error code for execution reverted, so just have to check the error string